src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1303,6 +1303,11 @@ protected static SyntaxList<TNode> RemoveRange<TNode>(SyntaxList<TNode> list, in
         public abstract SyntaxNode ThrowStatement(SyntaxNode expression = null);
 
         /// <summary>
+        /// Creates an expression that can be used to throw an exception.
+        /// </summary>
+        public abstract SyntaxNode ThrowExpression(SyntaxNode expression);
```

Comment:
alright, wasn't aware of that.  Will this be merged in before VB throw expressions are done?